metricbeat/module/kibana/stats/_meta/data.json

```diff
@@ -0,0 +1,69 @@
+{
+    "@timestamp": "2018-04-19T23:17:10.298Z",
+    "beat": {
+        "hostname": "host.example.com",
+        "name": "host.example.com"
+    },
+    "kibana": {
+        "stats": {
+            "cluster_uuid": "G279hqGeSDqjj_OgBq6wUw",
```

Comment:
If this is the elasticsearch cluster uuid, we should put it under `elasticsearch.cluster.id` as we do in other events. This makes it possible to correlate all events from one cluster. I can take care of that in a follow up PR.

Reply:
It would affect the Kibana PR as well as this one, so wouldn't it make sense to do this before merging, to reduce the follow-up needed?

The Kibana PR still depends on another one that no one has had time to review yet, so we still have some time to adjust the data structure.

Reply:
Correct me if I'm wrong about this affecting the Kibana PR. Maybe as an alternative the solution is to map incoming `cluster_uuid` value to `elasticsearch.cluster.id` in `metricbeat/module/kibana/stats/data.go`? If yes, then I'd agree to taking care of it in a followup PR.